docker-compose.yml

```diff
@@ -28,6 +28,13 @@ services:
       - PGDATA=/var/lib/postgresql/data/pgdata
     env_file:
       - .env
+    
+  cronjob:
+    image: appropriate/curl
```

Comment:
We need to pull an image that has curl or wget with CA installed, defaul alpine wget fail to work because no certificates are installed...